pkgs/wlroots/metadata.nix

```diff
@@ -1,5 +1,8 @@
-{
-  repo_git = "https://gitlab.freedesktop.org/wlroots/wlroots";
+rec {
+  domain = "gitlab.freedesktop.org";
+  owner = "wlroots";
+  repo = "wlroots";
```

Comment:
opinions on moving these to here?

Reply:
it's definitely in line with what I want, and looks similar to another approach I'm going to try to use in a couple places for rpi updates in nixpkgs...